lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -359,6 +370,26 @@ instance FromJSON WalletPutData where
 instance ToJSON  WalletPutData where
     toJSON = genericToJSON defaultRecordTypeOptions
 
+instance FromJSON PostExternalTransactionData where
+    parseJSON = genericParseJSON defaultRecordTypeOptions
+instance ToJSON PostExternalTransactionData where
+    toJSON = genericToJSON defaultRecordTypeOptions
+
+instance ToJSON ByteString where
```

Comment:
maybe we need to hide this stuff in `instance ToJSON PostExternalTransactionData` to avoid orphan instances coming from this :thinking: